modules/account-lib/test/unit/coin/sol/transactionBuilder/stakingDeactivateBuilder.ts

```diff
@@ -0,0 +1,144 @@
+import should from 'should';
+
+import { register } from '../../../../../src';
+import { TransactionBuilderFactory, KeyPair, Utils } from '../../../../../src/coin/sol';
+import * as testData from '../../../../resources/sol/sol';
+
+describe('Sol Staking Deactivate Builder', () => {
+  const factory = register('tsol', TransactionBuilderFactory);
+
+  const wallet = new KeyPair(testData.authAccount).getKeys();
+  const stakeAccount = new KeyPair(testData.stakeAccount).getKeys();
+  const recentBlockHash = 'GHtXQBsoZHVnNFa9YevAzFr17DJjgHXk3ycTKD5xD3Zi';
+  const invalidPubKey = testData.pubKeys.invalidPubKeys[0];
+
+  describe('Should succeed', () => {
```

Comment:
lets add coverage for verifying fees. also, i we need to include tests for `explainTransaction` (can be done in a separate PR)

Reply:
STLX-12452 Has been created to address this.